### ISSUE TITLE
Backend API:

### ISSUE THREAD

**Issue by JoshLmao** [5CS024-Team1/asset-tracker-web#18]
Replace reusable code in a nice helper file/class

- [ ] Parsing db row

- [x]  API calls placed in helper file

- [x]  Database column names put in helper

**Comment by JoshLmao** [comment]
Per the "Parsing db row", I've seen this used multiple times around the backend and when I changed one column, I had to go around figuring out where it was used and replacing it. 

https://github.com/5CS024-Team1/asset-tracker-web/blob/bd0bca458ed0c8ba021e1e7431794cb4b6dcb688/api/assets/get/index.php#L38-L50

**Comment by ghost** [comment]
Yeah something that I've noticed as well with the user, reports and assets page is that they also have some reusable code across them
